python/oksconfgen/generate_readoutOKS.py

```diff
@@ -101,24 +113,41 @@ def generate_readout(readoutmap, oksfile, include, segment, session):
     rogs = db.get_dals(class_name="ReadoutGroup")
     hermes_controllers = db.get_dals(class_name="HermesController")
 
-    if len(db.get_dals(class_name="LatencyBuffer")) > 0:
+    if len(db.get_dals(class_name="LatencyBuffer")) > 0 and tpg_enabled:
```

Comment:
This is so that if TPG is disabled, we don't use the existing ReadoutModuleConfs but generate new ones with TPG disabled (https://github.com/DUNE-DAQ/oksconfgen/blob/8d5c87581d3dd20c77c651a7ab2f2f3b5d7de866/python/oksconfgen/generate_readoutOKS.py#L145)